lib/rubycritic/commands/base.rb

```diff
@@ -9,7 +9,7 @@ def initialize(options)
       end
 
       def execute
-        fail NotImplementedError
+        raise NotImplementedError
```

Comment:
Not the biggest fan of this since I really like the semantic difference between "fail" and "raise". 
However, sticking a reasonable style guide project wide is more important than my personal taste here.
@Onumis @lucasmazza wdyt?
If you guys don't care we can just go with "raise" for all-the-things.